### PR TITLE
351: Venue creation form

### PR DESCRIPTION
**Pull request by maxb94**

### CODE REVIEW

File: src/pages/Account/Account.scss [hunk 1]
```diff
@@ -74,6 +74,40 @@ $login-max-width: 540px;
...
+}
+
+#admin-venue .input-container {
+  margin-top: 20px;
+}
+
+#admin-venue .input-title {
+  font-weight: bold;
+  margin-bottom: 5px;
+}
```
Comment (antoineeudes) [inline src/pages/Account/Account.scss:94]: #admin-venue {
   .input-title {
         ...
    }

   .input-container {
         ...
    }
    ...
}

File: src/pages/Account/Account.scss [hunk 2]
```diff
@@ -74,6 +74,40 @@ $login-max-width: 540px;
...
+#admin-venue .input-logo-container {
+  width: 150px;
+  height: 150px;
+}
+
+#admin-venue .input-logo-image {
+  height: 100%;
+  object-fit: cover;
+}
+
```
Comment (thecadams) [inline src/pages/Account/Account.scss:110]: Same as @antoineeudes comments applies

File: src/pages/Account/AdminVenue.tsx [hunk 3]
```diff
@@ -1,10 +1,173 @@
-import React from "react";
+import React, { useMemo } from "react";
 import WithNavigationBar from "components/organisms/WithNavigationBar";
+import { useForm, FieldError } from "react-hook-form";
+import * as Yup from "yup";
+
+const LONG_DESCRIPTION_PLACEHOLDER =
+  "Describe what is unique andwonderful and sparkling about your venue";
```
Comment (thecadams) [inline src/pages/Account/AdminVenue.tsx:7]: tpyo: andwonderful

File: src/pages/Account/AdminVenue.tsx [hunk 4]
```diff
@@ -1,10 +1,186 @@
...
+  name: string;
+  bannerImageUrl: FileList;
+  logoImageUrl: FileList;
+  tagline: string;
+  longDescription: string;
+}
+
+const validationSchema = Yup.object().shape<FormInputs>({
+  name: Yup.string()
+    .required("Display name required")
```
[the remark this answers is not in the document]
Comment (maxb94) [reply src/pages/Account/AdminVenue.tsx:21]: we should. This ticket has gone on long enough.

**Pull request merged**